src/DnMultiSelect/DnMultiSelect.css

```diff
@@ -0,0 +1,169 @@
+/**
+ * Copyright 2020, Verizon Media
```

Comment:
I'm not sure if we can change it to Yahoo. please check.

Reply:
@jazminorozco Can you advice?